src/librustc_mir/transform/add_yk_swt_calls.rs

```diff
@@ -18,6 +18,11 @@ use rustc::hir;
 use rustc::hir::def_id::{DefIndex, LOCAL_CRATE};
 use rustc::hir::map::blocks::FnLikeNode;
 
+// Crates the trace recorder depends upon.
+// This is effectively libstd and its dependencies (from its Cargo manifest).
+static RECORDER_DEPS: &'static [&'static str] = &["std", "alloc", "panic_unwind",
```

Comment:
I think it might be clearer to call this something like `CRATES_TRACE_BLACKLIST`: in a sense, the fact that the crate is a dependency is less relevant than that we're going to avoid tracing it?

Reply:
OK.